app/src/main/java/cz/cvut/fjfi/pvs/pvs2016/util/JSONUtils.java

```diff
@@ -0,0 +1,45 @@
+package cz.cvut.fjfi.pvs.pvs2016.util;
+
+import static cz.cvut.fjfi.pvs.pvs2016.util.FileUtils.getMetadataFiles;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+/**
+ * Created by fnj on 6.12.16.
+ */
+
+public class JSONUtils {
+	private static JSONArray getMetadataRootArray(File jsonFile) {
+		JSONParser parser = new JSONParser();
+		JSONArray rootArray = null;
+		try {
+			rootArray = (JSONArray) parser.parse(new FileReader(jsonFile));
+		} catch (ParseException | IOException e) {
+			e.printStackTrace();
+		}
+		return rootArray;
+	}
+
+	public static ArrayList<Photo> getPhotoList(ArrayList<Photo> photoList) {
```

Comment:
nechat pouze funkci bez parametru, prázný list není potřeba přidávat, i kdyby měl již někdo existující list a chtěl do něj přidat další objekty Photo, tak to nebude dělat tímto způsobem